src/main.ts

```diff
@@ -19,6 +19,9 @@ const internalExternsFileName = 'tsickle_externs.js';
 interface Settings {
   /** If provided, path to save externs to. */
   externsPath?: string;
+
+  /** If provided, convert every type to the Closure {?} type */
+  isTyped: boolean;
```

Comment:
The comment seems to say the opposite of what the variable name indicates, maybe just a typo.

I think this would be better expressed as an optional "untyped" field, so that it can default to falsey (either unprovided or false).  At least, that's how it works in the underlying tsickle library.